src/compiler/diagnosticMessages.json

```diff
@@ -5357,6 +5357,14 @@
         "category": "Message",
         "code": 95066
     },
+    "Add missing call": {

```

Comment:
"Add missing argument list" or "Add missing call parentheses"?

Reply:
+1 for the latter